spec/controllers/newflow/base_controller_spec.rb

```diff
@@ -27,6 +27,36 @@ module Newflow
       end
     end
 
+    describe 'POST #sheerid_webhook' do
+      let(:handler) { SheeridWebhook }
+
+      let(:params) do
+        {
```

Comment:
imo, this should be one line.  stuff like this creates lots and lots of vertical space and over time, gets harder to read

Reply:
improved.